pkg/metabase/client.go

```diff
@@ -331,7 +337,7 @@ func (c *Client) AddPermissionGroupMember(ctx context.Context, groupID int, emai
 	return c.request(ctx, http.MethodPost, "/permissions/membership", payload, nil)
 }
 
-func (c *Client) RestrictAccessToDatabase(ctx context.Context, groupID, databaseID int) error {
+func (c *Client) RestrictAccessToDatabase(ctx context.Context, groupIDs []int, databaseID int) error {
```

Comment:
If I understand correctly this is sort of the same answer as the previous review comment. The way it is implemented now from nada-backend there is a 1:1 mapping between a dataset in markedsplassen (i.e. table in bigquery) and a metabase database. This could obviously be a problem if one were to manually setup databases in metabase linked to multiple bigquery tables, but we are not allowing this for the users primarily due to it requiring admin access to metabase.